packages/yoastseo/src/values/Mark.js

```diff
@@ -108,6 +109,36 @@ Mark.prototype.serialize = function() {
 	};
 };
 
+/**
+ * Checks if the mark object is valid.
+ * @returns {void}
+ */
+// eslint-disable-next-line complexity
```

Comment:
I don't know how to make this any less complex. So for now ignore it.

Reply:
I also cannot think of a way to simplify it 😅 . So yes we can ignore it for now 👍🏽